text-generation-inference/server/text_generation_server/tgi_service.py

```diff
@@ -0,0 +1,29 @@
+import os
+from pathlib import Path
+from loguru import logger
+import sys
+from text_generation_server import server
+import argparse
+
+
+def main(args):
+    logger.info("TGIService: starting tgi service .... ")
```

Comment:
Does `TGIService` refer to a specific class or so?

Reply:
the class to be started by deepspeed for triggering server.serve